test/functional/puppetclasses_controller_test.rb

```diff
@@ -14,13 +14,14 @@ def test_edit
 
   def test_update_invalid
     Puppetclass.any_instance.stubs(:valid?).returns(false)
-    put :update, {:id => Puppetclass.first.to_param}, set_session_user
+    put :update, {:id => Puppetclass.first.to_param, :puppetclass => {:name => nil}}, set_session_user
     assert_template 'edit'
   end
 
   def test_update_valid
     Puppetclass.any_instance.stubs(:valid?).returns(true)
-    put :update, {:id => Puppetclass.first.to_param}, set_session_user
+    put :update, {:id => Puppetclass.first.to_param, :puppetclass => {:name => "Mrs. Piggy"}}, set_session_user
+    assert "Mrs, Piggy", Puppetclass.first.name
```

Comment:
In fact tests should've been red due to this one at least, note the `Mrs DOT` vs `Mrs COMMA`.